build.gradle

```diff
@@ -46,6 +46,7 @@ buildscript {
 ext {
     groupId = 'de.cyface'
     version = "5.0.0-beta1"
+    artifactId = 'android-backend'
```

Comment:
This was added accidentally and is removed in one of the last commits. The artifactIds of the submoduls which are published to the Github Package Registry is generated in the publish.gradle file.